docs/en/stack/ml/df-analytics/flightdata-classification.asciidoc

```diff
@@ -0,0 +1,477 @@
+[role="xpack"]
+[testenv="platinum"]
+[[flightdata-classification]]
+=== Predicting delaying flights with {classanalysis}
+
+Let's try to predict whether a flight will be delayed or not by using the 
+{kibana-ref}/add-sample-data.html[sample flight data]. We use this data to learn 
+a {classification} model that allows us to make the prediction based on 
+information such as weather conditions and location of the destination and 
+origin, flight distance and carrier.
+
+This dataset consists of sample data that has been manually created and it 
+contains inconsistencies. For example, a flight can be both delayed and 
+canceled. Nonetheless, the use case is relevant and it is a good example of how 
+the quality of input data will affect the quality of results.
+
+We create a {dfanalytics-job} to predict if a given flight is delayed. This 
+information is recorded as a boolean true/false variable on the field 
+`FlightDelay`. Each document in the dataset contains details for a single 
+flight, so this data is ready for analysis as it is already in a two-dimensional 
+entity-based data structure (_{dataframe}_). In general, you often need to 
+{ref}/transforms.html[transform] the data into an entity-centric index before 
+you analyze the data.
+
+This is an example source document from the dataset:
+
+```
+{
+  "_index": "kibana_sample_data_flights",
+  "_type": "_doc",
+  "_id": "S-JS1W0BJ7wufFIaPAHe",
+  "_version": 1,
+  "_seq_no": 3356,
+  "_primary_term": 1,
+  "found": true,
+  "_source": {
+    "FlightNum": "N32FE9T",
+    "DestCountry": "JP",
+    "OriginWeather": "Thunder & Lightning",
+    "OriginCityName": "Adelaide",
+    "AvgTicketPrice": 499.08518599798685,
+    "DistanceMiles": 4802.864932998549,
+    "FlightDelay": false,
+    "DestWeather": "Sunny",
+    "Dest": "Chubu Centrair International Airport",
+    "FlightDelayType": "No Delay",
+    "OriginCountry": "AU",
+    "dayOfWeek": 3,
+    "DistanceKilometers": 7729.461862731618,
+    "timestamp": "2019-10-17T11:12:29",
+    "DestLocation": {
+      "lat": "34.85839844",
+      "lon": "136.8049927"
+    },
+    "DestAirportID": "NGO",
+    "Carrier": "ES-Air",
+    "Cancelled": false,
+    "FlightTimeMin": 454.6742272195069,
+    "Origin": "Adelaide International Airport",
+    "OriginLocation": {
+      "lat": "-34.945",
+      "lon": "138.531006"
+    },
+    "DestRegion": "SE-BD",
+    "OriginAirportID": "ADL",
+    "OriginRegion": "SE-BD",
+    "DestCityName": "Tokoname",
+    "FlightTimeHour": 7.577903786991782,
+    "FlightDelayMin": 0
+  }
+}
+```
+
+{classification-cap} is a supervised machine learning analysis and therefore 
+needs to train on data that contains the ground truth, known as the 
+`dependent_variable`. In this example, the ground truth is available in each 
```

Comment:
I think at this point we're referring to the concept of dependent variables, not to a specific entity:
```suggestion
_dependent variable_. In this example, the ground truth is available in each 
```